Assets/HotmaxEditorVR/Hittables/propTile.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class propTile : MonoBehaviour, IHittable
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+
+    public int currentPage;
```

Comment:
unless im missing something you aren't using in your class. 
Which means you can remove it here, and our of your `displayPropInMenu` argument, and out from the constructor invocation in loadPropsMenu

Reply:
I think we need it here because the Start() on propTile is where the current page needs to be referenced.  That is the only place telling the propTile to load the correct model in the menu

Reply:
I am going to change the name to currentSpawnPage

Reply:
i don't understand what you mean... you are passing this into your class from propsMenu - to pass it into your `displayPropInMenu` method as an argument.. but then you don't use it in that method at all

Reply:
got it